### DISCUSSION TITLE
Spa Heat Boost

### DISCUSSION THREAD

**Discussion by bp529** [tagyoureit/nodejs-poolController#935] [Ideas]
I found this feature on an old Compool 2000 system and it was really useful. I'm thinking this might be best achieved for Nixie using something like Node Red, but figured I'd ask here as well. 

This feature basically raises the set point of the spa by an offset for a short period of time (5min) and then returns the set point back. It's enough to kick the heater on and give you a quick boost of heat without fiddling with set points. It was mostly implemented on spa side remotes.

**Comment by tagyoureit** [comment]
We have the Heat Boost defined as a pump circuit, but only because it is defined in the OCP's.  It would need to be wired up in the UI to be triggered and then some logic to change the set point temporarily.

![image](https://github.com/tagyoureit/nodejs-poolController/assets/1646840/852a81ed-dd57-45d6-bfd2-813dd3c6b0c4)
